snapcraft/hooks/common

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh -eu
```

Comment:
It's unlikely to happen in this case, but it's usually best to have `set -eu` on a separated line to also have those flags applied if someone is to call `sh ./script`.